nfc_ros/CMakeLists.txt

```diff
@@ -27,12 +27,22 @@ catkin_generate_virtualenv(
     PYTHON_INTERPRETER python3
     CHECK_VENV FALSE
 )
-catkin_install_python(PROGRAMS
-  node_scripts/nfc_ros_node.py
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-)
+file(GLOB NODE_SCRIPTS_FILES node_scripts/*.py)
```

Comment:
This changes the installed directory, beacsue we prefer to have same direcotry structure between devel and installed @sktometometo